src/client/menu/menu.c

```diff
@@ -2322,6 +2406,19 @@ LoadGame_MenuKey(int key)
         LoadGame_MenuInit();
         return menu_move_sound;
 
+    case K_DEL:
```

Comment:
Do we can also attach some button for joystick? What do you think? @Yamagi @DanielGibson

Reply:
dunno..
Can the existing savegame menu functionality be used via joystick?
Is there an "obvious" joystick button for this? It would suck if people accidentally deleted their savegames if pressing the wrong button on the joystick.. (Yes, there's still the confirmation dialogue, but that would need to be joystick-navigable as well, so I fear that with one fatfingered press onto the controller one could accidentally delete and confirm)